cli/internal/core/engine.go

```diff
@@ -422,3 +431,45 @@ func (e *Engine) GetTaskGraphDescendants(taskID string) ([]string, error) {
 	sort.Strings(stringDescendents)
 	return stringDescendents, nil
 }
+
+func (e *Engine) getPipelineFromWorkspace(workspaceName string) (*fs.Pipeline, error) {
+	cachedPipeline, ok := e.pipelines[workspaceName]
+	if ok {
+		return cachedPipeline, nil
+	}
+
+	// TODO(mehulkar) what is this value when it's a non-root workspace?
```

Comment:
Is this TODO still needed?

Reply:
removed it. i know the answer now, but it's still something to grapple with when I want to look up configs in other, non-root workspaces.